app/src/main/java/org/p2p/wallet/root/RootPresenter.kt

```diff
@@ -1,18 +1,30 @@
 package org.p2p.wallet.root
 
-import org.p2p.wallet.common.mvp.BasePresenter
-import org.p2p.wallet.feerelayer.interactor.FeeRelayerAccountInteractor
-import org.p2p.wallet.sell.interactor.SellInteractor
-import org.p2p.wallet.swap.interactor.orca.OrcaInfoInteractor
 import timber.log.Timber
+import kotlinx.coroutines.Deferred
 import kotlinx.coroutines.async
 import kotlinx.coroutines.awaitAll
 import kotlinx.coroutines.launch
+import org.p2p.wallet.common.feature_toggles.toggles.remote.BooleanFeatureToggle
+import org.p2p.wallet.common.feature_toggles.toggles.remote.NewSwapEnabledFeatureToggle
+import org.p2p.wallet.common.feature_toggles.toggles.remote.SellEnabledFeatureToggle
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.feerelayer.interactor.FeeRelayerAccountInteractor
+import org.p2p.wallet.sell.interactor.SellInteractor
+import org.p2p.wallet.swap.interactor.orca.OrcaInfoInteractor
+import org.p2p.wallet.swap.jupiter.repository.routes.JupiterSwapRoutesRepository
+import org.p2p.wallet.swap.jupiter.repository.tokens.JupiterSwapTokensRepository
+
+private const val TAG = "RootPresenter"
```

Comment:
maybe we can use class name
in case it could be renamed 🙂

Reply:
Presenter converts to T / F / S from RootPresenter::class.simpleName when using proguard